src/gpu/d3d11/SDL_gpu_d3d11.c

```diff
@@ -5787,6 +5788,19 @@ static bool D3D11_SupportsTextureFormat(
         }
     }
 
+    // Checks for SIMULTANEOUS_READ_WRITE support
+    if (usage & SDL_GPU_TEXTUREUSAGE_COMPUTE_STORAGE_SIMULTANEOUS_READ_WRITE) {
```

Comment:
Are we starting down the path of features partially supported and needing to be queried by the app?

Reply:
I guess that's always been the case for texture types and usages?

Reply:
The _feature_ is universally supported, but the specific _formats_ you can use with it have to be checked. Texture support already requires conditional checks. We have a list of universally supported formats for different use cases in the documentation for SDL_GPUTextureFormat, but some issues can't be avoided. For example, some hardware uses D24_S8 and some uses D32_S8. The client just has to check which one they have to use.